### PR TITLE
feat(bq,pg|h3,quadbin): center as default mode in H3_POLYFILL and QUADBIN_POLYFILL

### PR DESCRIPTION
**Pull request by Jesus89**
# Description

Shortcut

- Story: https://app.shortcut.com/cartoteam/story/335419
- Autolink: [sc-335419]

Set 'center' as the default mode for H3_POLYFILL and QUADBIN_POLYFILL. Updated docs and tests accordingly.

## Type of change

- Feature
- Tests
- Documentation

### CODE REVIEW

File: clouds/bigquery/modules/doc/quadbin/QUADBIN_POLYFILL_MODE.md [hunk 1]
```diff
@@ -6,14 +6,18 @@ QUADBIN_POLYFILL_MODE(geog, resolution, mode)
...
+Returns an array of quadbin cell indexes contained in the given geography at a given level of detail. Containment is determined by the mode: center, intersects, contains.
 
-* `geog`: `GEOGRAPHY` representing the area to cover.
+* `geog`: `GEOGRAPHY` representing the shape to cover.
 * `resolution`: `INT64` level of detail. The value must be between 0 and 26.
 * `mode`: `STRING`
-  * `intersects` (default) returns the indexes of the quadbin cells that intersect the input geography. The resulting quadbin set will completely cover the input geography (point, line, polygon). This mode is the one used in [`QUADBIN_POLYFILL`](quadbin#quadbin_polyfill).
+  * `center` returns the indexes of the quadbin cells which centers intersect the input geography (polygon). The resulting quadbin set does not fully cover the input geography, however, this is **significantly faster** that the other modes. This mode is not compatible with points or lines. Equivalent to [`QUADBIN_POLYFILL`](quadbin#quadbin_polyfill).
```
Comment (vdelacruzb) [inline clouds/bigquery/modules/doc/quadbin/QUADBIN_POLYFILL_MODE.md:14]: same

File: clouds/bigquery/modules/doc/h3/H3_POLYFILL_MODE.md [hunk 2]
```diff
@@ -6,14 +6,18 @@ H3_POLYFILL_MODE(geog, resolution, mode)
...
+Returns an array of quadbin cell indexes contained in the given geography at a given level of detail. Containment is determined by the mode: center, intersects, contains.
 
-* `geog`: `GEOGRAPHY` representing the area to cover.
+* `geog`: `GEOGRAPHY` representing the shape to cover.
 * `resolution`: `INT64` level of detail. The value must be between 0 and 15 ([H3 resolution table](https://h3geo.org/docs/core-library/restable)).
 * `mode`: `STRING`
-  * `intersects` (default) returns the indexes of the H3 cells that intersect the input geography. The resulting H3 set will completely cover the input geography (point, line, polygon). This mode is the one used in [`H3_POLYFILL`](h3#h3_polyfill).
+  * `center` returns the indexes of the H3 cells which centers intersect the input geography (polygon). The resulting H3 set does not fully cover the input geography, however, this is **significantly faster** that the other modes. This mode is not compatible with points or lines. Equivalent to [`H3_POLYFILL`](h3#h3_polyfill).
```
[the remark this answers is not in the document]
Comment (Jesus89) [reply clouds/bigquery/modules/doc/h3/H3_POLYFILL_MODE.md:14]: There is no default in this function

File: clouds/bigquery/modules/doc/quadbin/QUADBIN_POLYFILL_MODE.md (hunk 1 above)
Comment (Jesus89) [reply clouds/bigquery/modules/doc/quadbin/QUADBIN_POLYFILL_MODE.md:14]: https://github.com/CartoDB/analytics-toolbox-core/pull/439#discussion_r1277211484

File: clouds/bigquery/modules/doc/h3/H3_POLYFILL_MODE.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vdelacruzb) [reply clouds/bigquery/modules/doc/h3/H3_POLYFILL_MODE.md:14]: True, in BQ is mandatory to choose.

**Review by vdelacruzb — APPROVED** [review]
LGTM. I just updated the title so it's easier to identify when generating changelogs

**Pull request merged**